benchmarks/nemo/nemolite2d/manual_versions/regent/Makefile

```diff
@@ -0,0 +1,16 @@
+F90 = gfortran
+
+%.o : %.f90
+	$(F90) -fPIC -c -o $@ $<
+
+all: libgocean2d_io_mod.so
+regent: nemolite2d.exe
+
+libgocean2d_io_mod.so : kind_params_mod.o gocean2d_io_mod.o
+	$(F90) kind_params_mod.o gocean2d_io_mod.o -shared -o $@
+
+nemolite2d.exe: 
```

Comment:
Shouldn't this have all *.rg as target dependencies or be a PHONY target?

Reply:
Done